packages/cli-kit/src/private/node/ui/hooks/use-select-state.ts

```diff
@@ -320,7 +345,7 @@ export const useSelectState = <T>({visibleOptionCount, options, defaultValue}: U
   }, [])
 
   const visibleOptions = useMemo(() => {
-    return options.slice(state.visibleFromIndex, state.visibleToIndex)
+    return options.slice(state.visibleFromIndex)
```

Comment:
Do we still need visibleToIndex if we don't use it?

Reply:
It's still used [here](https://github.com/Shopify/cli/pull/2099/files#diff-a2d15153e3c654b299aede1768af71ce7b182c9d5bb26e669affdd5f591ce30eR129).

Reply:
In that case, could we remove the dependency on it in this useMemo?